rust/src/applayer.rs

```diff
@@ -126,9 +136,14 @@ pub struct AppLayerTxData {
     /// not using application layer keywords
     pub guessed_applayer_logged: u8,
 
-    /// detection engine flags for use by detection engine
-    detect_flags_ts: u64,
-    detect_flags_tc: u64,
+    /// detection engine progress tracking for use by detection engine
+    /// Reflects the "progress" of prefilter engines into this TX, where
+    /// the value is offset by 1. So if for progress state 0 the engines
+    /// are done, the value here will be 1. So a value of 0 means, no
+    /// progress tracked yet.
```

Comment:
This part is a bit confusing to me:
     /// So if for progress state 0 the engines
    /// are done, the value here will be 1. So a value of 0 means, no
    /// progress tracked yet.

Reply:
yes